spec/lib/resque-serializer/serializers/both_spec.rb

```diff
@@ -21,107 +21,66 @@ def self.perform(*args); end
 end
 
 RSpec.describe JobSerializedByBoth do
-  let(:args) { %w(arg1 arg2) }
+  let(:args)       { %w[arg1 arg2] }
+  let(:queue_name) { :default }
+
+  subject(:execute_job) { worker.reserve }
```

Comment:
Let's move this to the `ResqueTestHelper` module. It might be a little weird to not have a `subject` in the specs, but at least all the helper-type logic will be together.